Makefile

```diff
@@ -1,12 +1,36 @@
+# NOTE: this requires a kernel with tools/lib/bpf bulit (for libbpf.so)
+LINUX_SOURCE=../linux
+# ^^ is there a better way to do this??
```

Comment:
I don't see a way to do this that is simpler for a dev than the current without being slower (downloading relevant bits in the makefile) or more messy (requiring a specific subfolder of the kernel)